### ISSUE TITLE
Problem with object composition

### ISSUE THREAD

**Issue by cihati** [kristerkari/stylelint-react-native#18]
```
const baseStyle = {
  inboxContentContainer: {
    flex: 1
  },
... # irrelevant
}

const unreadContainerStyle = {
  ...baseStyle.mainContainer,
  ...baseStyle.containerUnread
}

const isTypingStyle = {
  ...baseStyle.messagePreview,
  ...baseStyle.green
}

module.exports = StyleSheet.create({
  ...baseStyle,
  unreadContainerStyle,
  isTypingStyle
})
```

I'm getting

```
Unexpected unknown property "unread-container-style"   react-native/style-property-no-unknown
Unexpected unknown property "unread-container-style"   react-native/css-property-no-unknown
Unexpected unknown property "is-typing-style"          react-native/style-property-no-unknown
Unexpected unknown property "is-typing-style"          react-native/css-property-no-unknown
```

errors for such code.  my .stylelintrc is:

```
{
  "plugins": ["stylelint-react-native"],
  "rules": {
    "react-native/style-property-no-unknown": true,
    "react-native/css-property-no-unknown": true
  }
}
```

any ideas?

**Comment by kristerkari** [comment]
Thanks! 

I'm not sure how stylelint handles the styles that are defined inside an Javascript object, but I'll have a look at it.

**Comment by kristerkari** [comment]
There is a fix available in version 2.0.2: 
- https://github.com/kristerkari/stylelint-react-native/releases/tag/2.0.2

Make sure that you only enable `react-native/style-property-no-unknown` when using React Native's `StyleSheet` API.
